target/linux/rockchip/image/mmc.bootscript

```diff
@@ -1,8 +1,15 @@
 part uuid mmc ${devnum}:2 uuid
 
-setenv bootargs "console=ttyS2,1500000 console=tty1 earlycon=uart8250,mmio32,0xff1a0000 root=PARTUUID=${uuid} rw rootwait"
+if test $stdout = 'serial@ff130000' ;
+then setenv bootargs "console=ttyS2,1500000 console=tty1 earlycon=uart8250,mmio32,0xff130000 swiotlb=1 root=PARTUUID=${uuid} rw rootwait loglevel=9";
+else setenv bootargs "console=ttyS2,1500000 console=tty1 earlycon=uart8250,mmio32,0xff1a0000 swiotlb=1 root=PARTUUID=${uuid} rw rootwait loglevel=9";
+fi;
 
 load mmc ${devnum}:1 ${fdt_addr_r} rockchip.dtb
 load mmc ${devnum}:1 ${kernel_addr_r} kernel.img
 
-booti ${kernel_addr_r} - ${fdt_addr_r}
+#set new address and decompress lzma kernel
+setexpr kernel_addr_dec ${filesize} + ${kernel_addr_r}
```

Comment:
Please use
`setexpr kernel_addr_dec ${filesize} + ${kernel_addr_r} || kernel_addr_dec=0x03000000`

Some rk33xx platforms (like my PBP) ship with the rockchip bsp u-boot on their SPI NOR flash and will use it instead of the u-boot in OpenWRT. This u-boot doesn't support setexpr, but we can just fallback on this safe address.

Reply:
on the RockPro64 with RK3399:
```
=> echo $stdout
serial@ff1a0000
=> 
```
earlycon works!

Reply:
added both tests because if the address is not correct there will be no output and misleading that the board is not booting